elements/pfe-progress-steps/src/pfe-progress-steps.js

```diff
@@ -53,14 +53,19 @@ class PfeProgressSteps extends PFElement {
 
   constructor() {
     super(PfeProgressSteps, { type: PfeProgressSteps.PfeType });
+    this._resizeObserver = new ResizeObserver(this._build.bind(this));
   }
 
   connectedCallback() {
     super.connectedCallback();
-    this._build();
+    // this will call _build initially and estabilish a resize observer for this element
+    this._resizeObserver.observe(this);
   }
 
-  disconnectedCallback() {}
+  disconnectedCallback() {
+    super.connectedCallback();
```

Comment:
Probably a bad idea to call the connectedCallback in the disconnectedCallback

Reply:
This could be a "life comes at you fast meme". 🤦‍♂️

Reply:
![cover4](https://user-images.githubusercontent.com/3428964/132007668-9caa847e-25ec-4789-a376-63bc4d2ce7bc.jpeg)

Reply:
super command fixed.